worlds/noita/Items.py

```diff
@@ -40,12 +39,14 @@ def create_all_items(world: MultiWorld, player: int) -> None:
         itempool += ["Perk (Spatial Awareness)"]
 
     # Create any non-progression repeat items (referred to as junk regardless of whether it's useful)
-    junk_pool = item_pool_weights[pool_option]
+    filler_pool = filler_weights
```

Comment:
```suggestion
    filler_pool = filler_weights.copy()
```

Else it will modify the original which could cause unintended consequences.